qml/js/yleApi.js

```diff
@@ -108,7 +108,7 @@ var api = (function(appId, appKey) {
                                ? program.partOfSeason.seasonNumber
                                : "",
                  episodeNumber: program.episodeNumber ? program.episodeNumber : "",
-                 seriesId: program.partOfSeason && program.partOfSeason.id,
+                 seriesId: program.partOfSeries && program.partOfSeries.id

```

Comment:
so `partOfSeason` is then something that we can use later on and find programs by season?

Reply:
We already use `program.partOfSeason.seasonNumber `a few lines above, this was probably a bug.

Reply:
:+1: